docs/study-def-tricks.md

```diff
@@ -353,3 +353,35 @@ In addition:
   instead, it saves the joined extracts in a new output directory.
   Replacing the extracts makes it harder to construct an audit trail, which reduces computational and analytical transparency;
   [core principles](index.md) of the OpenSAFELY platform.
+
+## Grouping IMD into quintiles
+
+---8<-- 'includes/imd-warning-header.md'
+
+To group IMD into quintiles, then:
+
+```py
+study = StudyDefinition(
+    # ...
+    imdQ5 = patients.categorised_as(
+        {
+            "0": "DEFAULT",
+            "1": "imd >=0 AND imd < 32844*1/5",
```

Comment:
Should we add a note that 32844 is the total LSOAs at the time of the IMD rankings currently in TPP (2019 data I think); it could change with the next update, and study defs would need to update this number

Reply:
62c9814 replaces approximate values with actual values for both original and rounded rankings. I've also linked to [_English indices of deprivation 2019_](https://www.gov.uk/government/statistics/english-indices-of-deprivation-2019), which describes the number of LSOAs in England.